scripts/update-custodian.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+usage()
+{
+  echo -e "usage: ./update-custodian.sh phase register custodian\ne.g. ./update-custodian.sh alpha country \"Jane Doe\""
+}
+
+# validation check number of args but other validation is done in python script
+if [ $# -lt 3 ]; then
+  echo "error: not enough arguments"
+  usage
+  exit 1
+fi
+
+PHASE=$1
+REGISTER=$2
+CUSTODIAN=$3
+echo "PHASE - $PHASE"
+echo "REGISTER - $REGISTER"
+echo "CUSTODIAN - $CUSTODIAN"
+
+OPENREGISTER_BASE="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+echo "OPENREGISTER_BASE - $OPENREGISTER_BASE"
+source "$OPENREGISTER_BASE/deployment/scripts/includes/register-actions.sh"
+
+TIME=`date +%Y-%m-%dT%H:%M:%SZ`
+JSON="{\"custodian\":\"$CUSTODIAN\"}"
+SHA256=`echo -n $JSON | shasum -a 256 | head -c 64`
+PASSWORD=`PASSWORD_STORE_DIR=~/.registers-pass pass $PHASE/app/mint/$REGISTER`
```

Comment:
Does this override any env variable you might have in your environment? If so, should we respect the environment?

Reply:
fixed in 2ccaa815b41e24935c7f35d60f8f9f9db7c18c77